2025-10-13 00:28:59,033 [INFO] django.utils.autoreload autoreload:667: Watching for file changes with StatReloader
2025-10-13 00:29:11,084 [INFO] matplotlib.font_manager font_manager:1639: generated new fontManager
2025-10-13 00:32:56,103 [INFO] django.utils.autoreload autoreload:667: Watching for file changes with StatReloader
2025-10-13 00:33:06,305 [WARNING] django.request log:253: Not Found: /
2025-10-13 00:33:06,306 [WARNING] django.server basehttp:213: "GET / HTTP/1.1" 404 2662
2025-10-13 00:33:06,624 [WARNING] django.request log:253: Not Found: /favicon.ico
2025-10-13 00:33:06,626 [WARNING] django.server basehttp:213: "GET /favicon.ico HTTP/1.1" 404 2713
2025-10-13 01:24:05,341 [INFO] django.utils.autoreload autoreload:667: Watching for file changes with StatReloader
2025-10-13 01:24:08,951 [WARNING] django.request log:253: Not Found: /
2025-10-13 01:24:08,952 [WARNING] django.server basehttp:213: "GET / HTTP/1.1" 404 2662
2025-10-13 01:28:49,550 [WARNING] django.request log:253: Not Found: /ws/updates/
2025-10-13 01:28:49,560 [WARNING] django.request log:253: Unauthorized: /api/results/
2025-10-13 01:28:49,561 [WARNING] django.server basehttp:213: "GET /ws/updates/ HTTP/1.1" 404 2713
2025-10-13 01:28:49,561 [WARNING] django.server basehttp:213: "GET /api/results/ HTTP/1.1" 401 58
2025-10-13 01:28:49,563 [WARNING] django.request log:253: Unauthorized: /api/results/
2025-10-13 01:28:49,564 [WARNING] django.server basehttp:213: "GET /api/results/ HTTP/1.1" 401 58
2025-10-13 01:28:57,446 [WARNING] django.request log:253: Unauthorized: /api/results/
2025-10-13 01:28:57,451 [WARNING] django.request log:253: Not Found: /ws/updates/
2025-10-13 01:28:57,451 [WARNING] django.server basehttp:213: "GET /api/results/ HTTP/1.1" 401 58
2025-10-13 01:28:57,452 [WARNING] django.server basehttp:213: "GET /ws/updates/ HTTP/1.1" 404 2713
2025-10-13 01:28:57,454 [WARNING] django.request log:253: Unauthorized: /api/results/
2025-10-13 01:28:57,456 [WARNING] django.server basehttp:213: "GET /api/results/ HTTP/1.1" 401 58
2025-10-13 01:29:06,178 [WARNING] django.request log:253: Unauthorized: /api/upload/
2025-10-13 01:29:06,179 [WARNING] django.server basehttp:213: "POST /api/upload/ HTTP/1.1" 401 58
2025-10-13 01:33:10,162 [WARNING] django.request log:253: Unauthorized: /api/upload/
2025-10-13 01:33:10,163 [WARNING] django.server basehttp:213: "POST /api/upload/ HTTP/1.1" 401 58
2025-10-13 01:36:18,135 [INFO] django.utils.autoreload autoreload:265: C:\Users\Mr\Documents\SIH-Prototype-CIRCAD\circad\backend\circad_backend\settings.py changed, reloading.
2025-10-13 01:36:19,226 [INFO] django.utils.autoreload autoreload:667: Watching for file changes with StatReloader
2025-10-13 01:36:49,102 [INFO] django.utils.autoreload autoreload:265: C:\Users\Mr\Documents\SIH-Prototype-CIRCAD\circad\backend\api\views.py changed, reloading.
2025-10-13 01:36:50,266 [INFO] django.utils.autoreload autoreload:667: Watching for file changes with StatReloader
2025-10-13 01:37:22,293 [INFO] django.utils.autoreload autoreload:265: C:\Users\Mr\Documents\SIH-Prototype-CIRCAD\circad\backend\api\views.py changed, reloading.
2025-10-13 01:37:23,636 [INFO] django.utils.autoreload autoreload:667: Watching for file changes with StatReloader
2025-10-13 01:37:28,952 [INFO] django.utils.autoreload autoreload:265: C:\Users\Mr\Documents\SIH-Prototype-CIRCAD\circad\backend\api\views.py changed, reloading.
2025-10-13 01:37:30,100 [INFO] django.utils.autoreload autoreload:667: Watching for file changes with StatReloader
2025-10-13 01:40:08,835 [INFO] django.server basehttp:213: "POST /api/upload/ HTTP/1.1" 202 161
2025-10-13 01:40:09,519 [INFO] django.server basehttp:213: "POST /api/analyze/1/ HTTP/1.1" 200 461
2025-10-13 01:40:15,585 [WARNING] django.request log:253: Not Found: /ws/updates/
2025-10-13 01:40:15,586 [WARNING] django.server basehttp:213: "GET /ws/updates/ HTTP/1.1" 404 2713
2025-10-13 01:40:15,598 [INFO] django.server basehttp:213: "GET /api/results/ HTTP/1.1" 200 463
2025-10-13 01:40:15,623 [INFO] django.server basehttp:213: "GET /api/results/ HTTP/1.1" 200 463
2025-10-13 01:40:19,449 [WARNING] django.request log:253: Not Found: /ws/updates/
2025-10-13 01:40:19,450 [WARNING] django.server basehttp:213: "GET /ws/updates/ HTTP/1.1" 404 2713
2025-10-13 01:40:19,465 [INFO] django.server basehttp:213: "GET /api/results/ HTTP/1.1" 200 463
2025-10-13 01:40:19,494 [INFO] django.server basehttp:213: "GET /api/results/ HTTP/1.1" 200 463
